pages/fiscal-sponsorship/mobile/index.js

```diff
@@ -0,0 +1,101 @@
+import { Box, Container, Heading } from 'theme-ui'
+import {
+  PillHolder,
+  AuthorPill,
+  DatePill
+} from '../../../components/announcements/pills'
+import Head from 'next/head'
+import NextLink from 'next/link'
+import styled from '@emotion/styled'
+import theme from '../../../lib/theme'
+import Meta from '@hackclub/meta'
+import Nav from '../../../components/nav'
+import ForceTheme from '../../../components/force-theme'
+import Footer from '../../../components/footer'
+import Copy from '../../../components/announcements/hcb-mobile.mdx'
+import SlackCTA from '../../../components/announcements/cta'
+import AnnouncementHolder from '../../../components/announcements/holder'
+import Balancer from 'react-wrap-balancer'
+
+const StyledLink = styled.a`
+  text-decoration: underline;
+  color: ${theme.colors.white};
+`
+
+const Link = props => {
+  const { href } = props
+  return (
+    <NextLink href={href} passHref>
+      <StyledLink>{props.children}</StyledLink>
+    </NextLink>
+  )
+}
+
+const RelonPage = () => (
+  <>
+    <Meta
+      as={Head}
+      title="HCB Mobile is here!"
+      description="View and manage your HCB organizations on the go. Issue cards, view transactions, and more!"
```

Comment:
"View" twice here felt repetitive.
```suggestion
      description="Manage your HCB organizations on the go. Issue cards, view transactions, and more!"
```